internal/pkg/service/common/servicectx/servicectx.go

```diff
@@ -0,0 +1,155 @@
+// Package servicectx provides unique ID for a service process and support for the graceful shutdown.
+package servicectx
```

Comment:
Potreboval som nejaku abstrakciu pre unikatne ID daneho procesu API/Worker, ktore pouzijem v `etcd elections`.

Tak som spravil `servicectx.Process` (nic lepsie mi nenapadlo).

Rovno som sem pridal `OnShutdown` pre riešenie graceful shutdown, ktoré bude potrebné na viacerých miestach vo Worker, a takto to bude bezpecnejsie a citatelnejsie.